### ISSUE TITLE
It won't open file -path

### ISSUE THREAD

**Issue by Budweizer65** [goatfungus/NMSSaveEditor#935]
I also tried renaming and deleting the NMSSaveEditor.conf file.
![goat error](https://github.com/goatfungus/NMSSaveEditor/assets/47543090/2343e5c8-af70-41c6-ac37-f12c12bfe946)
![goat_nms](https://github.com/goatfungus/NMSSaveEditor/assets/47543090/b0fda60b-2157-4bd2-8be0-8de8132229b3)

**Comment by cjg67** [comment]
The editor is working perfectly since the author has updated it to account for the changes made by hello games.  
I am successfully using it to make changes, even while I am still completing an expedition.
You need to make sure that you have CORRECTLY installed the LATEST version of java, and also make sure that you have correctly installed the latest version of the editor.
A number of issues reported on here about the editor, are user errors only, and not true bugs at all.

You also make no mention of which platform you are using, which makes this kind of a report absolutely useless to the developer and a complete waste of there time.  You need to learn to actually put complete details into any report you make.  This is one of the main reasons that free projects eventually close down.  Why should developers have to wade through so much crap because of some self absorbed asshole who can not be bothered giving complete details.

**Comment by Budweizer65** [comment]
" Why should developers have to wade through so much crap because of some self absorbed asshole who can not be bothered giving complete details."

I'm not sure where you get calling people asshole's for reporting a issue they are having. No one said it was the software issue or mine. I did not say their software was shit or worthless. If you go through other people's reports, there have been others with the same issue ( from the last version ). I have used this program for years without any issues. The only asshole here in this thread so far is you. But thank you for taking the time out of your day to check out this report.

BTW when you called me or someone else Self Absorbed just shows your ignorance and you were most likely describing yourself.
Self-absorbed "means absorbed in one's own thoughts, activities, or interests. It is an adjective that can be used to describe someone who is egocentric, narcissistic, or solipsistic." <-- That sir is pretty much you.
